tests/framework/installer/ceph_installer.go

```diff
@@ -152,27 +154,21 @@ func (h *CephInstaller) startAdmissionController() error {
 	return nil
 }
 
-// CreateRookOperatorViaHelm creates rook operator via Helm chart named local/rook present in local repo
-func (h *CephInstaller) CreateRookOperatorViaHelm(chartSettings string) error {
-	// creating clusterrolebinding for kubeadm env.
-	h.k8shelper.CreateAnonSystemClusterBinding()
-
-	// create the operator namespace before the admission controller is created
-	if err := h.k8shelper.CreateNamespace(h.settings.OperatorNamespace); err != nil {
-		return errors.Errorf("failed to create namespace %s. %v", h.settings.Namespace, err)
-	}
-	if err := h.startAdmissionController(); err != nil {
-		return errors.Errorf("failed to start admission controllers: %v", err)
+func (h *CephInstaller) waitForToolboxPod(namespace string) (string, error) {
+	if err := h.k8shelper.WaitForLabeledPodsToRun("app=rook-ceph-tools", namespace); err != nil {
+		return "", errors.Wrap(err, "Rook Toolbox couldn't start")
 	}
-	if err := h.helmHelper.InstallLocalRookHelmChart(h.settings.Namespace, chartSettings); err != nil {
-		return errors.Errorf("failed to install rook operator via helm, err : %v", err)
-	}
-	// create the cluster namespace since it wasn't created by common.yaml
-	if err := h.k8shelper.CreateNamespace(h.settings.Namespace); err != nil {
-		return errors.Errorf("failed to create namespace %s. %v", h.settings.Namespace, err)
+	logger.Infof("Rook Toolbox started")
+
+	podNames, err := h.k8shelper.GetPodNamesForApp("rook-ceph-tools", namespace)
+	assert.NoError(h.T(), err)
+	for _, podName := range podNames {
+		// All e2e tests should run ceph commands in the toolbox since we are not inside a container
+		logger.Infof("found active toolbox pod: %q", podName)
+		return "", nil
```

Comment:
Looks like the tests are failing because this pod name isn't being set in `RunAllCephCommandsInToolboxPod`
```suggestion
		return podName, nil
```